plugins/camera/camera_impl.cpp

```diff
@@ -107,13 +106,31 @@ void CameraImpl::disable()
     _parent->remove_call_every(_flight_information_call_every_cookie);
 }
 
+Camera::Result CameraImpl::select_camera(unsigned id)
+{
+    if (id > 6) {
```

Comment:
We support Ids's 0 to 5 right ? If so, it should be 
```suggestion
    const size_t MAX_SUPPORTED_IDS = 5; // inside class
    if (id > MAX_SUPPORTED_IDS) {
```

Reply:
Good catch! I'll fix it.